cosmic-settings/src/pages/desktop/panel/inner.rs

```diff
@@ -502,11 +503,54 @@ impl PageInner {
         }
 
         if panel_config.anchor_gap || !panel_config.expand_to_edges {
-            panel_config.border_radius = 8;
+            let radii = Self::get_corner_radii().radius_xl[0] as u32;
```

Comment:
`cosmic::theme::system_preference().cosmic().corner_radii` should work here. I don't think it requires a new method.

Reply:
Awesome! I'll make the change.